packages/backend/src/rates/service.ts

```diff
@@ -107,13 +113,38 @@ class RatesServiceImpl implements RatesService {
     const url = this.deps.pricesUrl
     if (!url) return {}
 
-    const res = await this.axios.get(url).catch((err) => {
+    const res = await this.axios.get<PricesResponse>(url).catch((err) => {
       this.deps.logger.warn({ err: err.message }, 'price request error')
       throw err
     })
-    this.pricesRequest = Promise.resolve(res.data)
+
+    const { base, rates } = res.data
+    this.checkBaseAsset(base)
+
+    const data = {
+      [base]: 1.0,
+      ...(rates ? rates : {})
+    }
+
+    this.pricesRequest = Promise.resolve(data)
     this.pricesExpiry = new Date(Date.now() + this.deps.pricesLifetime)
-    return res.data
+    return data
+  }
+
+  checkBaseAsset(asset: unknown): void {
```

Comment:
```suggestion
  checkBaseAsset(asset: unknown): asset is string {
```
Even though `PricesResponse` already gives you the proper type, this (optionally) could be a good addition too

Reply:
```suggestion
  checkBaseAsset(asset: unknown): asset is string {
```

Isn't this covered by this check ? 👇
https://github.com/interledger/rafiki/blob/dc4a698b46af14e66501858da262919fb2d83387/packages/backend/src/rates/service.ts#L138-L139
By typing the return type as `asset is string` we will have to return and an extra check will be required: 
```ts
if(!this.checkBaseAsset(base)){
  // log & throw
}
```